### PR TITLE
Tools #2

### PR DESCRIPTION
**Pull request by iprice73**

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -1,6 +1,6 @@
-#include <vector>
+#include "functions.hpp"
```
Comment (Marcinati) [inline src/main.cpp]: It should be the last header here.

https://coders.school/edu/courses/powtorka-1-qa/lesson/kolejnosc-include/

**Review by Kasperice — APPROVED** [review]
Looks good

File: src/main.cpp [hunk 2]
```diff
@@ -1,6 +1,6 @@
-#include <vector>
 #include <iostream>
-#include "../inc/functions.hpp"    // TODO: add proper compilation flag and change to #include "functions.hpp"
+#include <vector>
+#include "functions.hpp"
```
Comment (DamiXioR) [inline src/main.cpp:3]: move it up

File: Makefile [hunk 3]
```diff
@@ -1,3 +1,3 @@
 all: src/*.cpp inc/*.hpp
-	g++ -std=c++17 src/*.cpp -o main
+	g++ -std=c++17 src/*.cpp -o main -Werror -Wall -Wextra -pedantic -Wconversion -Iinc/
```
Comment (DamiXioR) [inline Makefile:2]: in my opinion looks better: -I./inc

**Review by DamiXioR — COMMENTED** [review]
some ideas

File: src/main.cpp (hunk 2 above)
Comment (Marcinati) [reply src/main.cpp:3]: It's good that way according to Mateusz:

https://coders.school/edu/courses/powtorka-1-qa/lesson/kolejnosc-include/

Other example:

https://github.com/coders-school/kurs_cpp_podstawowy/blob/module2/module1/homework/fibonacci/main.cpp

**Comment by ziobron** [comment]
+4 XP Narzędzia#2 – vimtutor
+5 XP Narzędzia#2 – git - konflikty
+4 XP Narzędzia#2 – flagi kompilacji
+3 XP Narzędzia#2 – git - cherrypick
+8 XP delivery before 07.06.2020

{ "requirement_ids": [1119, 1999, 1119, 2000, 1119, 2001, 1119, 2002], "points": [2, 4, 2, 5, 2, 4, 2, 3] }

**Pull request closed without merging**